src/controllers/user.controller.ts

```diff
@@ -11,6 +11,22 @@ interface UserAuthData {
 export class UserController {
   constructor(private userService: UserService) {}
 
+  public async readAllUsers(req: Request, res: Response): Promise<Response> {
+    try {
+      const { _id } = req.user!;
+
+      const users = await userService.getUsers();
+
+      if (!users) {
+        res.status(StatusCodes.BAD_REQUEST).send();
+      }
+
+      return res.send({ users });
```

Comment:
What is the result here? Is it `{[{...}, {...}, {...}]}` or ``[{...}, {...}, {...}]` (should be the second one)